wallets/rn_cli_wallet/package.json

```diff
@@ -15,47 +16,63 @@
     "postinstall": "patch-package"
   },
   "dependencies": {
+    "@craftzdog/react-native-buffer": "6.0.5",
     "@json-rpc-tools/utils": "1.7.6",
+    "@notifee/react-native": "7.8.2",
     "@react-native-async-storage/async-storage": "1.21.0",
     "@react-native-clipboard/clipboard": "1.13.2",
     "@react-native-community/netinfo": "11.1.0",
-    "@react-navigation/bottom-tabs": "^6.5.11",
+    "@react-native-firebase/app": "18.8.0",
+    "@react-native-firebase/messaging": "18.8.0",
+    "@react-navigation/bottom-tabs": "6.5.11",
+    "@react-navigation/elements": "1.3.21",
     "@react-navigation/native": "6.1.9",
+    "@react-navigation/native-stack": "6.9.17",
     "@react-navigation/stack": "6.3.20",
     "@sentry/react-native": "5.17.0",
+    "@walletconnect/notify-client": "0.16.5",
+    "@walletconnect/notify-message-decrypter": "^0.1.0",
     "@walletconnect/react-native-compat": "2.11.0",
     "@walletconnect/web3wallet": "1.10.0",
+    "dayjs": "^1.11.10",
     "ethers": "5.7.2",
+    "lodash.clonedeep": "^4.5.0",
     "patch-package": "^8.0.0",
     "postinstall-postinstall": "2.1.0",
+    "expo": "^49.0.0",
     "react": "18.2.0",
-    "react-native": "0.73.2",
+    "react-hook-form": "^7.50.1",
```

Comment:
are you using this lib?

Reply:
Yes. See `SubscriptionsSettingsScreen`. RHF is a great lib when compared with the others (like Formic). It has quite simple usage here but I'm happy to keep it still.